scripts/set/phising/send_attack.py

```diff
@@ -2,79 +2,101 @@
 '''
 This is the command-line front-end in charge of processing arguments and call the framework
 '''
-import getopt, sys, os
+
+import getopt
+import sys
+import os
 import subprocess
 from collections import defaultdict
 
+
 def GetName():
-	return sys.argv[0]
+    return sys.argv[0]
+
 
 def Usage(Message):
-	print "Usage:"
-	print GetName()+" EMAIL_TARGET=? EMAIL_FROM=? SMTP_LOGIN=? SMTP_PASS=? SMTP_HOST=? SMTP_PORT=? EMAIL_PRIORITY=? PDF_TEMPLATE=? MSF_LISTENER_PORT=? MSF_LISTENER_SETUP=? ATTACHMENT_NAME=? SET_EMAIL_TEMPLATE=? PHISHING_PAYLOAD=? PHISHING_SCRIPT_DIR=? TOOL_SET_DIR=?"
-	print "ERROR: "+Message
-	sys.exit(-1)
+    print "Usage:"
+    print GetName()+" EMAIL_TARGET=? EMAIL_FROM=? SMTP_LOGIN=? SMTP_PASS=?" + \
+        " SMTP_HOST=? SMTP_PORT=? EMAIL_PRIORITY=? PDF_TEMPLATE=? MSF_LISTENER_PORT=? " + \
+        "MSF_LISTENER_SETUP=? ATTACHMENT_NAME=? SET_EMAIL_TEMPLATE=? PHISHING_PAYLOAD=? " + \
+        "PHISHING_SCRIPT_DIR=? TOOL_SET_DIR=?"
+    print "ERROR: "+Message
+    sys.exit(-1)
+
 
 def ShellExec(Command):
-	print "\nExecuting (Control+C to abort THIS COMMAND ONLY):\n"+Command
-	Output = ''
-	try: # Stolen from: http://stackoverflow.com/questions/5833716/how-to-capture-output-of-a-shell-script-running-in-a-separate-process-in-a-wxpyt
-		proc = subprocess.Popen(Command, shell=True,
-			stdout=subprocess.PIPE,
-			stderr=subprocess.STDOUT,
-			bufsize=1)
-		while True:
-			line = proc.stdout.readline()
-			if not line: break
-			print MultipleReplace(line, { "\n":"", "\r":"" }) # Show progress on the screen too!
-			Output += line # Save as much output as possible before a tool crashes! :)
-	except KeyboardInterrupt:
-		Output += self.Core.Error.UserAbort('Command', Output) # Identify as Command Level abort
-	return Output
+    print "\nExecuting (Control+C to abort THIS COMMAND ONLY):\n" + Command
+    Output = ''
+    # noqa Stolen from: http://stackoverflow.com/questions/5833716/how-to-capture-output-of-a-shell-script-running-in-a-separate-process-in-a-wxpyt
+    try:
+        proc = subprocess.Popen(Command, shell=True,
+                                stdout=subprocess.PIPE,
+                                stderr=subprocess.STDOUT,
+                                bufsize=1)
+        while True:
+            line = proc.stdout.readline()
+            if not line:
+                break
+            print MultipleReplace(line, {"\n": "", "\r": ""})  # Show progress on the screen too!
+            Output += line  # Save as much output as possible before a tool crashes! :)
+    except KeyboardInterrupt:
+        Output += self.Core.Error.UserAbort('Command', Output)  # Identify as Command Level abort
```

Comment:
@delta24 Hi, found a bug. The `Output += self.Core.Error...` will throw an error as there is no self defined (its a function). What do you suggest should be done here ?